### PR TITLE
Daytime deploy script adjustments

### PR DESCRIPTION
**Pull request by samueljmello**

### CODE REVIEW

File: deployment/iisDeployApplication.bat [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+SET deploymentFolder=%1
+SET applicationSymlink=%2
+
+ECHO Validating variables
+IF [%deploymentFolder%] == [] GOTO missingDeploymentFolder
+IF [%applicationSymlink%] == [] GOTO missingApplicationSymlink
+
+ECHO Validating deployment folder
+IF NOT EXIST %deploymentFolder% GOTO missingDeploymentFolder
+IF NOT EXIST %applicationSymlink% GOTO missingApplicationSymlink
```
Comment (charlie-retzler) [inline deployment/iisDeployApplication.bat]: If symlink doesn't exists can't we just create it and not error out?

File: deployment/iisDeployCleanUp.bat [hunk 2]
```diff
@@ -0,0 +1,38 @@
+@ECHO off
+CLS
+
+ECHO Getting variables from command input
+SET deploymentsFolder=%1
```
Comment (charlie-retzler) [inline deployment/iisDeployCleanUp.bat]: Is this a different deployment folder than the previous script? Possibly the parent folder? If so can we name them so it makes it obvious they are different?

File: deployment/iisDeployApplication.bat [hunk 3]
```diff
@@ -0,0 +1,37 @@
+@ECHO off
```
Comment (charlie-retzler) [inline deployment/iisDeployApplication.bat]: Can we call this something like SwitchApplicationFoldersToLatestRelease?

File: deployment/iisDeployCleanUp.bat [hunk 4]
```diff
@@ -0,0 +1,38 @@
+@ECHO off
```
Comment (charlie-retzler) [inline deployment/iisDeployCleanUp.bat]: Can we rename this CleanupOldVersionsOfApplication?

File: deployment/iisDeployCleanUp.bat (hunk 2 above)
Comment (samueljmello) [inline deployment/iisDeployCleanUp.bat]: changed to newDeploymentFolder

File: deployment/iisDeployCleanUp.bat (hunk 4 above)
Comment (samueljmello) [inline deployment/iisDeployCleanUp.bat]: Done

**Pull request merged**